packages/statemanager/README.md

```diff
@@ -101,14 +136,20 @@ The `RPCStateManager` can be be used with any JSON-RPC provider that supports th
 
 In order to have an EVM instance that supports the BLOCKHASH opcode (which requires access to block history), you must instantiate both the `RPCStateManager` and the `RpcBlockChain` and use that when initalizing your EVM instance as below:
 
-```js
-import { RPCStateManager, RPCBlockChain } from '../src/rpcStateManager.js'
+```ts
+// ./examples/evm.ts
+
+import { RPCStateManager, RPCBlockChain } from '@ethereumjs/statemanager'
 import { EVM } from '@ethereumjs/evm'
 
-const blockchain = new RPCBlockChain({}, provider)
-const blockTag = 1n
-const state = new RPCStateManager({ provider, blockTag })
-const evm = new EVM({ blockchain, stateManager: state })
+const main = async () => {
+  const provider = 'https://path.to.my.provider.com'
+  const blockchain = new RPCBlockChain(provider)
+  const blockTag = 1n
+  const state = new RPCStateManager({ provider, blockTag })
+  const evm = new EVM({ blockchain, stateManager: state })
```

Comment:
Likely we need some note around here that now EVM is ready to run BLOCKHASH opcodes (over RPC)

Reply:
Added.